styles/components/_answers.scss

```diff
@@ -0,0 +1,4 @@
+.answers {
+    display: flex;
+    flex-wrap: wrap;
+}
```

Comment:
Is it going to be bigger? If not then I don't see the reason for setting separate scss file